scrapy/core/downloader/handlers/ftp.py

```diff
@@ -33,12 +33,16 @@
 from six.moves.urllib.parse import urlparse
 
 from twisted.internet import reactor
-from twisted.protocols.ftp import FTPClient, CommandFailed
+from twisted.protocols.ftp import FTPClient, FTPFileListProtocol, CommandFailed
 from twisted.internet.protocol import Protocol, ClientCreator
 
 from scrapy.http import Response
+from scrapy.http.response.ftplist import FTPListResponse
 from scrapy.responsetypes import responsetypes
 
+class ListDataProtocol(FTPFileListProtocol):
+    pass
```

Comment:
Removed subclass in https://github.com/bernardotorres/scrapy/commit/1c862e1f6a442cb69ec0b307f08a336d660e4705